### PR TITLE
MNT Don't include tests in distribution code

### PR DESCRIPTION
**Pull request by maxime-rainville**
Fixes https://github.com/silverstripe/silverstripe-versioned-admin/issues/199

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/tests/
+/tests/ export-ignore
```
Comment (emteknetnz) [inline .gitattributes:1]: Note sure if it matters, though I think we may want this without the trailing slash similar to what admin has.  Guaranteed that it'll work without the trailing slash

https://github.com/silverstripe/silverstripe-admin/blob/1/.gitattributes#L1

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Minor recommended change

**Comment by emteknetnz** [comment]
Travis failure is behat on graphql4 job only https://github.com/silverstripe/silverstripe-versioned-admin/pull/200/checks?check_run_id=2053303830

File: .gitattributes (hunk 1 above)
Comment (maxime-rainville) [reply .gitattributes:1]:
```suggestion
/tests export-ignore
```

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**